Frameworks/GLKit/GLKTexture.mm

```diff
@@ -277,7 +279,8 @@ @implementation GLKTextureLoader {
 */
 + (GLKTextureInfo*)textureWithContentsOfFile:(NSString*)fname options:(NSDictionary*)opts error:(NSError**)err {
     CGDataProviderRef provider = CGDataProviderCreateWithFilename([fname UTF8String]);
-    CGImageRef img = CGImageCreateWithPNGDataProvider(provider, NULL, NO, kCGRenderingIntentDefault);
+    unsigned char* dataBytes = static_cast<unsigned char*>(const_cast<void*>(_CGDataProviderGetData(provider)));
+    CGImageRef img = _CGImageGetImageFromData(dataBytes, _CGDataProviderGetSize(provider));
```

Comment:
was planning on then switched back, adding.

---
In reply to: [103335956](https://github.com/Microsoft/WinObjC/pull/2106#discussion_r103335956) [](ancestors = 103335956)